src/cox.jl

```diff
@@ -123,6 +123,9 @@ StatsAPI.vcov(obj::CoxModel) = obj.vcov
 
 StatsAPI.stderror(obj::CoxModel) = sqrt.(diag(vcov(obj)))
 
+StatsAPI.confint(obj::CoxModel; level::Real=0.95) = 
+  hcat(coef(obj),coef(obj)) + stderror(obj)*quantile(Normal(),(1. -level)/2.)*[1. -1.]
```

Comment:
```suggestion
function StatsAPI.confint(obj::CoxModel; level::Real=0.95)
    β = coef(obj)
    se = stderror(obj)
    q = quantile(Normal(), (1 - level) / 2)
    ci = β .+ se .* q
    return hcat(ci, .-ci)
end
```
There are a few separate reasons for this suggestion:
- This reads a bit clearer by breaking it up into assignments rather than a one-liner
- You can compute the quantities you need once to get the lower bound then simply negate them to get the upper bound rather than doing a series of array multiplications
- The style used by this package (and the majority of others, though not all) is 4 spaces for indentation rather than 2
- `1.` and `2.` are `Float64` literals, so `(1.0 - level) / 2.0` will force the result to `Float64` (assuming `level` isn't wider, e.g. `BigFloat`). Instead, you can do `(1 - level) / 2`, which uses `Int` literals and will preserve the type of `level`. Compare e.g. `(1.0 - 0.05f0) / 2.0` vs. `(1 - 0.05f0) / 2`: the former results in a `Float64` despite `level` being a `Float32` whereas the latter preserves `level`'s `Float32`. Not a big deal in this case tbh but it's generally considered good style to avoid float literals where integers would suffice.

Reply:
Great, 
I can definitely see what you mean. 
Thanks!

Reply:
Ha! Good catch, mine was definitely incorrect 😄